### DISCUSSION TITLE
How do you check built-in resource versions available?

### DISCUSSION THREAD

**Discussion by avoidik** [concourse/concourse#7107] [Help & Support] [answered]
Hi all,

Could you please suggest how to check Concourse standard resources versions, like docker-image or git, available for the deployment of specific version?

For example, I want to understand which version of [docker-image](https://github.com/concourse/docker-image-resource) is now available for my Concourse v5.6.0 deployment

Regards

**Comment by taylorsilva** [comment]
Great question! Our release notes now contain this info for our more recent versions under the "Bundled resource types" heading.

For older releases, like 5.6.0, where we didn't capture that info, you can manually verify by doing looking in the `resource_metadata.json` for each resource type. On a worker:
```
cat /usr/local/concourse/resource-types/<resource-name>/resource_metadata.json
```
the json looks like this:
```
{
  "type": "git",
  "version": "1.13.0",
  "privileged": false,
  "unique_version_history": false
}
```

**Reply by avoidik** [reply]
thank you @taylorsilva unfortunately, I have no access inside, anyway good to know

**Comment by chenbh** [comment, marked as answer]
The `/api/v1/workers` endpoint will report all the workers and their bundled resource types. I think you need to be logged in so you'll either have to do it from a browser or through `fly -t ci curl /api/v1/workers`

**Reply by taylorsilva** [reply]
or `fly -t ci workers --details --json`. Prints the resource versions for each worker.
